dll/cpl/timedate/internettime.c

```diff
@@ -319,6 +351,41 @@ OnInitDialog(HWND hwnd)
     CreateNTPServerList(hwnd);
 }
 
+static VOID
+OnAutoSync(BOOL Sync)
+{
+    HKEY hKey;
+    LONG lRet;
+    WCHAR szAuto[7];
+
+    if (Sync)
+        StringCbCopy(szAuto, wcslen(L"NTP"), L"NTP");
+    else
+        StringCbCopy(szAuto, wcslen(L"NoSync\0"), L"NoSync\0");
```

Comment:
Not needed. Just do:
```c
// Note: I keep the same name to not change all the namings elsewhere,
// but if one respects the code conventions it would be something like: lpszAuto ...
LPCWSTR szAuto;

if (Sync)
    szAuto = L"NTP";
else
    szAuto = L"NoSync";
```
This allows one to use constant strings without need to use temporary buffers.
The RegSetValueExW call is OK.

Reply:
I will change. Thanks.